plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/manager/GetClientDefaultsAnswer.java

```diff
@@ -0,0 +1,79 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package com.cloud.agent.api.manager;
+
+import com.cloud.agent.api.Answer;
+
+import java.util.Map;
+
+public class GetClientDefaultsAnswer extends Answer {
+
+    private String _currentApiVersion;
+    private Integer _apiRetryCount;
+    private Long _apiRetryInterval;
+
+    public GetClientDefaultsAnswer(GetClientDefaultsCommand cmd, Map<String, Object> defaults) {
+        super(cmd);
+        this._currentApiVersion = (String) defaults.get("CURRENT_API_VERSION");
+        this._apiRetryCount = (Integer) defaults.get("DEFAULT_API_RETRY_COUNT");
+        this._apiRetryInterval = (Long) defaults.get("DEFAULT_API_RETRY_INTERVAL");
+    }
+
+    public GetClientDefaultsAnswer(GetClientDefaultsCommand cmd, Exception e) {
+        super(cmd, e);
+    }
+
+    public String getCurrentApiVersion() {
+        return _currentApiVersion;
+    }
+
+    public Integer getApiRetryCount() {
+        return _apiRetryCount;
+    }
+
+    public Long getApiRetryInterval() {
+        return _apiRetryInterval;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Implemented <code>equals</code> method for the <code>Answer</code> super class, calling it from subclass.